docs/data/advanced-components/overview.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Advanced Components - Overview
+---
+
+# Advanced Components - Overview
+
+<p class="description">A collection of fast and powerful components.</p>
+
+## Licenses
```

Comment:
This section is inspired by [Data Grid - Getting Started # Plans](https://mui.com/components/data-grid/getting-started/#plans) but reworked